middlewares/sticky_backend_handler.go

```diff
@@ -0,0 +1,274 @@
+package middlewares
+
+import (
+	"bufio"
+	"bytes"
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"crypto/sha256"
+	"encoding/base64"
+	"fmt"
+	"html/template"
+	"io"
+	"net"
+	"net/http"
+	"sort"
+	"strings"
+
+	"github.com/Masterminds/sprig"
+	"github.com/containous/traefik/healthcheck"
+	"github.com/containous/traefik/log"
+	"github.com/containous/traefik/types"
+	guuid "github.com/satori/go.uuid"
+	"github.com/serialx/hashring"
+	"github.com/vulcand/oxy/forward"
+)
+
+// StickinessParsed holds the parsed representation of a Stickiness object
+type StickinessParsed struct {
```

Comment:
Can we rename this to something more "pleasant", like `StickinessConfig` or something along those lines?  `StickinessParsed` just sounds very strange/weird... 😜